pkg/asset/agent/manifests/common.go

```diff
@@ -22,7 +22,9 @@ func getInfraEnvName(ic *agent.OptionalInstallConfig) string {
 func getPullSecretName(ic *agent.OptionalInstallConfig) string {
 	return ic.Config.ObjectMeta.Name + "-pull-secret"
 }
-
+func getArchitecture(ic *agent.OptionalInstallConfig) string {
```

Comment:
Unless this is used in multiple places, I wouldn't bother defining a function for it.

Reply:
I kept this function so all of the arm64-> aarch64 translation can happen in one place, but I'm open to moving that logic elsewhere.

Reply:
and amd64 -> x86_64 translation